### ISSUE TITLE
I'm thinking to put an automatic updater.

### ISSUE THREAD

**Issue by turric4n** [newpascal/fpcupdeluxe#72]
Hi!

This project is awesome! But misses for an automatic updater... Can I develop a simple updater?

Logger URL uses a non standard HTTP port... I can host the logger using HTTPS server if you want.

**Comment by LongDirtyAnimAlf** [comment]
You're welcome !

Please explain: do you mean an auto-updater for fpcupdeluxe itself ... to get notifications about new versions ?

Thanks for your offer about a https-server.
The logger is just a gimmick for now.
It runs in a very small AWS-tier. I have https avvailable (see batterybutcher site itself).
If logging becomes serious, I will do the  necessary things to have a nice and standard URL.

**Comment by turric4n** [comment]
Yes, basically an auto-updater who notify and download the lastest release of FPCUpdate.

**Comment by mdbs99** [comment]
What is done, the self-updater?

**Comment by LongDirtyAnimAlf** [comment]
Just the URL.
Auto-updater is more than a few hours of work ... ;-)

**Comment by mdbs99** [comment]
Yes, of course :)
Because that I didn't understand the "done" so fast.

Just a suggestion: Could you put the issue numbers in your commits? We can track very easy changes that belongs to the issue. Thanks!
